mobile/lib/features/trade/domain/order.dart

```diff
@@ -76,6 +76,8 @@ class Order {
         direction: bridge.Direction.Long,
         orderType: bridge.OrderType.market(),
         state: bridge.OrderState.Open,
-        creationTimestamp: 0);
+        creationTimestamp: 0,
+        orderExpiryTimestamp: 0
```

Comment:
Is this field needed for something?

Reply:
it can be used to detect expired orders from the orderbook. Not used now, but it wouldn't compile if I didn't change it ;/